_maps/map_files/BoxStation/BoxStation.dmm

```diff
@@ -43376,22 +43374,43 @@
 	pixel_y = -22
 	},
 /obj/machinery/atmospherics/pipe/simple/scrubbers/hidden,
+/obj/machinery/power/apc{
+	dir = 4;
+	name = "Toxins Lab APC";
```

Comment:
Isn't this the misc_lab APC?

Reply:
well,IDK.I didn't touched this.

Reply:
![image](https://user-images.githubusercontent.com/6209658/34995625-114a52f0-faa5-11e7-88f8-295c6522172a.png)
check this APC.